### PR TITLE
Pagination: removed background SVG support #908

### LINKED ISSUE #927: Breadcrumbs & Pagination: keyboard focus indicator obscured
The full keyboard focus outline should be visible.

Safari

Screen Shot 2019-11-22 at 11 35 52 AM

Chrome

Screen Shot 2019-11-22 at 11 36 03 AM

## Pagination

Screen Shot 2019-11-26 at 9 46 09 AM

### PR DESCRIPTION
**Pull request by ianmcburnie**
# Issue #908

PLEASE SQUASH.

Removed background SVG support from pagination, plus following related (kinda) updates/cleanup/refactoring:

* Removed the pagination-mixins.less. Replaced the useful, resusable parts of that with a `layout-mixins`.less. The idea being that we might identify common micro-patterns with Flexbox and move them here.
* Adjusted the margin top and bottom for pagination, breadcrumbs, carousel and notice (ensures sensible minimum space between other modules)
* Fixes #927 . Not a perfect fix. We need to be more careful with our use of `overflow: hidden` as it causes problems. I had to use `outline-offset` to get around this. Will need to revisit.
* Updated icon mixins with RTL section (I simply rotate the svg icons that are direction based)
* Added tests for RTL, font-size increase & color inherit for pagination
* Updated contributing guide
* Fixed formatting of tests for notice

As a heads up, incase anyone has missed this, the following tests are now considered baseline tests for all modules:

* RTL
* font-size increase (upto 200%)
* color inherit (to a certain degree, where applicable)
* variable support (coming soon)

Example:

Screen Shot 2019-12-11 at 10 37 07 AM

Put simply, all of these must pass. They also help sniff out some code smells. For example, the breadcrumbs currently does not pass the font-size increase test due to the `overflow: hidden` that is present.

Screen Shot 2019-12-11 at 10 36 30 AM

As a side note, I see that `overflow: hidden` is also causing issue for visibility of keyboard focus indicator in some cases, so we really need to be carefully with how and where we apply this style.

### CODE REVIEW

File: src/less/mixins/layout/layout-mixins.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.flex-items-centered() {
+    align-items: center;
```
Comment (agliga) [inline src/less/mixins/layout/layout-mixins.less]: This also needs a `display: flex` or `display: inline-flex` to work. I see it is added in the other parts of the code, but someone using this might forget to add the correct display. 
Maybe we should add an argument to set the display to flex and if the argument for this mixin is null then it will default to `flex` or something. That way this is a standalone mixin and doesn't require any display properties outside of it. 
IE:
```
.flex-items-centered(@display: flex) {
   display: @display;
   // Other props. 
}
```
And then call it like:
`.flex-items-centered(inline-flex)`

Comment (ianmcburnie) [reply src/less/mixins/layout/layout-mixins.less]: Yep, I like that idea. Will update.

Comment (ianmcburnie) [reply src/less/mixins/layout/layout-mixins.less]: Done.

**Comment by ianmcburnie** [comment]
@agliga Good now?
